doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-mortgages-example-proc.adoc

```diff
@@ -18,51 +18,67 @@ The package you select must contain all the required rule assets.
-. Click the second property cell,
+.. Click *amount*.
+.. Click *Insert Data Object* to map the data object field to the property header cell.
+.. Optional: Click *Clear selection* to remove the selected data object.
+. Click the second property header cell,
 .. From *Test Tools* panel, select and expand the *LoanApplication* data object.
-.. Click *deposit* and then *Add*.
-. Click the third property cell,
+.. Click *deposit*.
+.. Click *Insert Data Object*.
+.. Optional: Click *Clear selection* to remove the selected data object.
+. Click the third property header cell,
 .. From *Test Tools* panel, select and expand the *LoanApplication* data object.
-.. Click *lengthYears* and then *Add*.
+.. Click *lengthYears*
+.. Click *Insert Data Object*.
+.. Optional: Click *Clear selection* to remove the selected data object.
 . Right-click *LoanApplication* header cell and select *Insert column right*. A new *GIVEN* column to the right is created.
 . Click the new header cell,
 .. From *Test Tools* panel, select the *IncomeSource* data object.
-.. Click *Add* to map the data object to the header cell.
-. Click the property cell below *IncomeSource*,
+.. Click *Insert Data Object* to map the data object to the header cell.
+. Click the property header cell below *IncomeSource*,
 .. From *Test Tools* panel, select and expand the *IncomeSource* data object.
-.. Click *type* and then *Add* to map the data object field to the property cell.
+.. Click *type*.
+.. Click *Insert Data Object* to map the data object field to the property header cell.
+.. Optional: Click *Clear selection* to remove the selected data object.
 +
 You have now defined all the *GIVEN* column cells.
 +
 . Next, define the *EXPECT* column details,
 .. Click the cell named *INSTANCE 2* under the *EXPECT* column header.
 .. From *Test Tools* panel, select *LoanApplication* data object.
-.. Click *Add*.
-. To create properties of the data object, right-click the property cell and select *Insert column right* or *Insert column left* as required. Create two more property cells under the *EXPECT* column.
-. Click the first property cell,
+.. Click *Insert Data Object*.
+.. Optional: Click *Clear selection* to remove the selected data object.
+. To create properties of the data object, right-click the property header cell and select *Insert column right* or *Insert column left* as required. Create two more property cells under the *EXPECT* column.
+. Click the first property header cell,
 .. From *Test Tools* panel, select and expand the *LoanApplication* data object.
-.. Click *approved* and then *Add* to map the data object field to the property cell.
-. Click the second property cell,
+.. Click *approved*.
+.. Click *Insert Data Object* to map the data object field to the property header cell.
+.. Optional: Click *Clear selection* to remove the selected data object.
+. Click the second header property cell,
 .. From *Test Tools* panel, select and expand the *LoanApplication* data object.
```

Comment:
Minor suggestion - add article:

s/From Test Tools panel,/From the Test Tools panel,